test/extended/testdata/build-secrets/s2i-binary-dir/.s2i/bin/assemble

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Copy secrets into a location they can be output during image run
+
+mkdir -p "${HOME}/testsecret"
+if [ -f /tmp/secret1 ]; then
```

Comment:
We should always prefer to use `[[` over `[` -- [here's](http://mywiki.wooledge.org/BashFAQ/031) some explanation of why, but the gist of it is `[` has a lot of ways to bite you.